### DISCUSSION TITLE
Hopefully glob syntax will be supported

### DISCUSSION THREAD

**Discussion by soeasyjx** [ranyitz/qnm#129] [General]
demo:
```shell
npx qnm "@umijs/*"
```

**Comment by ranyitz** [comment]
Sounds like a good idea!

Let me suggest how I think this can be implemented:

1. qnm has a modules map that we can use to get the list of modules (see [match](https://github.com/ranyitz/qnm/blob/5a8885a544b3962397a7ea975537d8beef07d608/src/workspace/workspace.ts#L319)).
2. Understand that the pattern is a glob pattern via [isGlob](https://github.com/micromatch/is-glob) and call the glob functionality added to the workspace (similar to how `match` is added).
3. Implement the matching between the pattern and the in-memory list of modules via [micromatch](https://github.com/micromatch/micromatch).

I don't see how this can break any existing queries, so I think it will be okay to add.

@soeasyjx do you want to try implementing this?
